packages/terra-compact-interactive-list/src/CompactInteractiveList.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import styles from './CompactInteractiveList.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * String that will be used to identify the table. If multiple tables are on the same page, each table should have
+   * a unique id.
+   */
+  id: PropTypes.string.isRequired,
+
+  /**
+   * String that identifies the element (or elements) that labels the table.
+   */
+  ariaLabelledBy: PropTypes.string,
+
+  /**
+   * String that labels the table for accessibility. If ariaLabelledBy is specified, ariaLabel will not be used.
+   */
+  ariaLabel: PropTypes.string,
+
+  /**
+   * Boolean indicating whether or not the table columns should be displayed. Setting the value to false will hide the columns,
+   * but the voice reader will use the column header values for a11y.
+   */
+  hasColumnHeaders: PropTypes.bool,
+};
```

Comment:
is this prop correct?

Reply:
I edited the prop to default to false and edited description per that. Currently the plan is to not display the headers, but allow users to turn them on if they need them.